Changed path where service_schema.json is stored

DaveVoyles opened the pull request:
I've tried exporting it directly into the root directory, but it never actually gets saved. It seems that you must place it in the ./outputs folder, which will then appear in the in the outputs section of ML Workbench after each run.

I also cannot find any documentation about specifically **why** this is, but I've noticed that they do it on the [Iris Classification tutorial as well.](https://docs.microsoft.com/en-us/azure/machine-learning/preview/tutorial-classifying-iris-part-3)

azurecla commented:
Hi __@hvredevoort__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Byronr commented:
@hvredevoort Thanks for the update. @tysonn Please merge.

DaveVoyles committed:
Changed path where service_schema.json is stored

I've tried exporting it directly into the root directory, but it never actually gets saved. It seems that you must place it in the ./outputs folder, which will then appear in the in the outputs section of ML Workbench after each run.

I also cannot find any documentation about specifically **why** this is, but I've noticed that they do it on the Iris Classification tutorial as well.
